### DISCUSSION TITLE
Page rendering with @unresolved

### DISCUSSION THREAD

**Discussion by timathom** [Jinntec/Fore#146] [Q&A] [answered]
Where/how is `body[@unresolved]` used to control page rendering? Is this part of Fore, or is it an addition to the demo forms? I'd like to ensure that no static HTML content is displayed until all bindings have been calculated.

**Comment by JoernT** [comment, marked as answer]
the 'unresolved' attribute is a left-over from times when i still used Polymer which controlled this attribute (removing it when all components are attached).

Nowadays - and better - you should use 'fx-ready' class for that purpose. It is added on either `<body>` and `<fx-fore>` elements immediately before the `ready` event is dispatched. This makes sure that this class will be present only after Fore is fully initialized.

You could for instance fade in the page by adding this to your CSS:

```
body{
    opacity:0;
    transition:opacity 0.6s;
}
body.fx-ready{
    opacity:1;
}
```

Likewise you can scope it to just the `<fx-fore>` element - e.g. when you have several on the page and want to control a single one.
```
#myfore.fx-fore{...}
```


`unresolved` is deprecated and should be removed from the demos.
